sphinx_gallery/gen_rst.py

```diff
@@ -638,9 +638,10 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     save_notebook(example_nb, example_file.replace('.py', '.ipynb'))
     with codecs.open(os.path.join(target_dir, base_image_name + '.rst'),
                      mode='w', encoding='utf-8') as f:
-        example_rst += "**Total running time of the script:**" \
-                       " ({0: .0f} minutes {1: .3f} seconds)\n\n".format(
-                           time_m, time_s)
+        if time_elapsed >= gallery_conf["min_reported_time"]:
```

Comment:
is there any situation in which not forcing this to be a float will bite us?

Reply:
If somebody sets the min computation time to not a float then that will crash but what can you do?...  Obviously an explicit type check could be added, feel free to push one.

Reply:
I'd assume YAGNI and rely on the Python error message to be explicit enough

Reply:
ah I was just thinking to add a `float(gallery_conf["min_reported_time"])` in there as opposed to doing a check

Reply:
sure, that might give better errors in any case

Reply:
That (so support strings when we want a float) I think would be unpythonically wrong.

Reply:
If we eventually add it as a command-line option I think it would come in as `str`, though

Reply:
ok I'm gonna wrap it in float for now and we can make it error later if we want